project/frontend/src/components/auth/LoginButton.tsx

```
@@ -0,0 +1,30 @@
+      onClick={handleLogin}
+      className="px-4 py-2"
+      style={{
+        color:
+          theme === "dark"
+            ? colors.text.light.primary
+            : colors.text.dark.primary,
+        backgroundImage: `linear-gradient(to right, ${
+          colors.primary[theme === "dark" ? "dark" : "light"]
+        }, ${colors.secondary[theme === "dark" ? "dark" : "light"]})`,
+      }}
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)